tests/arch/arm/arm_wrap_func/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  arch.wrap_func.arm:
+    tags: arm wrap_func
+    arch_allow: arm
```

Comment:
No filtering needed? WIll this work as-is for AARC64 and Cortex-R?

Reply:
I have no idea where it works, have only tried nRF53. What's the best way to filter on cortex-m aarch32?

Reply:
You seem to have implemented this for Cortex-M.
So you should filter with CPU_CORTEX_M or
 ARMV6_M_ARMV8_M_BASELINE or ARMV7_M_ARMV8_M_MAINLINE
unless you intend to making it work for all AARC32. You can test it on the qemu-cortex-r5